source/docs/magento2/add-new-attribute.md

```diff
@@ -0,0 +1,187 @@
+---
+id: add-new-attribute
+title: Add a new attribute
+---
+
+Let's say we want to add a new element to display on a product's page (for instance, a product's rate). You will have to:
+
+1. Add the attribute's actual value to your database.
+2. Create a new module.
+3. Define the schema, implement the resolver and display it on screen.
+
+## Add a value to your database
+
+The process to do so may vary with the backend software you are using. For Magento 2 (used in this example), you can refer to [this](https://www.fastcomet.com/tutorials/magento2/product-attributes).
+
+## Create a new GraphQL module
+
+We want to display the `rate` of a product on the product page. To keep it simple, it will only consist in displaying a number, the value can be manually edited by the merchant. This feature will be stored in a new module called `mymodule` ; you could directly edit the files at `node_modules/front-commerce/path/to/something`, but this is not recommended since running `npm install` might erase any modification.
+Learn more about Front-Commerce's folder structure [here](/docs/essentials/extend-the-theme.html).
+
+```bash
+mkdir mymodule
+cd mymodule
+```
+
+In this example, the files we are going to override are located at `node_modules/front-commerce/src/web/theme/modules/ProductView/Synthesis/`, so will create in our `mymodule` folder the following directories. More details [here](/docs/essentials/extend-the-graphql-schema.html).
+
+```bash
+mkdir -p server/modules/acme
```

Comment:
```suggestion
mkdir -p mymodule/server/modules/ratings
```

Let's make explicit that this should live in `mymodule`. Plus, `acme` is actually not a good name for our extension since it's only made for adding `ratings` functionality to the GraphQL server. Naming it will help people navigate in their codebase.

And this is an opportunity to change the fact that `acme` was `chosen arbitrarily`.